control/controlclient/direct.go

```diff
@@ -942,6 +954,22 @@ func (c *Direct) sendMapRequest(ctx context.Context, isStreaming bool, nu Netmap
 		TKAHead:                 tkaHead,
 		ConnectionHandleForTest: connectionHandleForTest,
 	}
+
+	// If we have a hardware attestation key, sign the node key with it and send
+	// the key & signature in the map request.
+	if k := persist.AsStruct().AttestationKey; k != nil {
+		hwPub := key.HardwareAttestationPublicFromPlatformKey(k)
+		request.HardwareAttestationKey = hwPub
+
+		nkBytes := nodeKey.Raw32()
+		sig, err := k.Sign(nil, nkBytes[:], crypto.SHA256)
```

Comment:
I worry that this signature might be replayable if stolen.
Is there something more ephemeral we could sign that would be approximately unique per request? It could even be the current timestamp if nothing better is available.